packages/task/src/browser/task-terminal-widget-manager.ts

```diff
@@ -103,11 +104,11 @@ export class TaskTerminalWidgetManager {
                         terminal.taskConfig = taskConfig;
                         terminal.busy = true;
                     } else {
-                        this.notifyTaskFinished(terminal);
+                        this.notifyTaskFinished(terminal, true);
```

Comment:
I'm fine with being explicit and setting the reuse to `true` despite it being the default.